whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -321,6 +289,22 @@ private static Autnum createAutnumResponse(final RpslObject rpslObject) {
         return autnum;
     }
 
+    private static Autnum createAsBlockResponse(final RpslObject rpslObject) {
+        final Autnum autnum = new Autnum();
+
+        String key = rpslObject.getValueForAttribute(AttributeType.AS_BLOCK).toString();
+        String[] startAndEndKey = key.split("-");
+
+        autnum.setHandle(startAndEndKey[0]);
+        //TODO :check what should be the name
```

Comment:
Check the RFC's for the definition, or for an example response that includes this.